app/controllers/projects_controller.rb

```diff
@@ -37,7 +37,7 @@ def create
 
   # PATCH/PUT /projects/1
   def update
-    if @project.update(project_params)
+    if update_project.project.update(project_params)
```

Comment:
IDK, you don't seem to understand what's going on here. Have some rest maybe and try again.